exporter/datadogexporter/translate_traces.go

```diff
@@ -117,7 +117,7 @@ func aggregateTracePayloadsByEnv(tracePayloads []*pb.TracePayload) []*pb.TracePa
 // converts a Trace's resource spans into a trace payload
 func resourceSpansToDatadogSpans(rs pdata.ResourceSpans, calculator *sublayerCalculator, hostname string, cfg *config.Config) pb.TracePayload {
 	// get env tag
-	env := cfg.Env
+	env := utils.NormalizeTag(cfg.Env)
```

Comment:
Should this only happen on traces? Or should we apply it to metrics too? Should we apply it to all tags?

Reply:
sorry, i didnt see these for some reason. probably all tags but i have to confirm